src/com/owncloud/android/providers/FileContentProvider.java

```diff
@@ -154,6 +155,57 @@
                 ProviderTableMeta.OCSHARES_ACCOUNT_OWNER);
     }
 
+    // Projection of Capabilities table
```

Comment:
The Capabilities Table projection is required for  the query method of FileContentProvider for accessing to a row of the Capabilities Table.